### PR TITLE
Added a LambdaTest runner to execute the patrol test.

### PR DESCRIPTION
**Pull request by Gauravpandeyr**

### CODE REVIEW

File: packages/patrol/example/android/app/build.gradle
Comment (mateuszwojtczak) [inline packages/patrol/example/android/app/build.gradle:1]: let's remove this change

File: packages/patrol/example/android/app/src/androidTest/java/pl/leancode/patrol/example/MainActivityTest.java
Comment (mateuszwojtczak) [inline packages/patrol/example/android/app/src/androidTest/java/pl/leancode/patrol/example/MainActivityTest.java:1]: let's remove this change

File: packages/patrol/example/android/gradle.properties
Comment (mateuszwojtczak) [inline packages/patrol/example/android/gradle.properties:1]: let's remove this change

File: packages/patrol/example/android/settings.gradle
Comment (mateuszwojtczak) [inline packages/patrol/example/android/settings.gradle:1]: let's remove this change

File: packages/patrol/android/src/main/kotlin/pl/leancode/patrol/LambdaTestPatrolJUnitRunner.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import android.util.Log;
+import pl.leancode.patrol.contracts.PatrolAppServiceClientException;
+
+import java.net.Inet4Address;
+import java.net.NetworkInterface;
+import java.net.SocketException;
+import java.util.Enumeration;
+import java.util.Objects;
+
+public class LambdatestPatrolJUnitRunner extends PatrolJUnitRunner {
```
Comment (mateuszwojtczak) [inline packages/patrol/android/src/main/kotlin/pl/leancode/patrol/LambdaTestPatrolJUnitRunner.java:12]:
```suggestion
public class LambdaTestPatrolJUnitRunner extends PatrolJUnitRunner {
```

File: packages/patrol/android/src/main/kotlin/pl/leancode/patrol/LambdaTestPatrolJUnitRunner.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    @Override
+    public PatrolAppServiceClient createAppServiceClient() {
+        // Create client with a default constructor (localhost:8082) by default.
+        PatrolAppServiceClient client = new PatrolAppServiceClient();
+        waitForPatrolAppService();
+
+        try {
+            client.listDartTests();
+
+            //TODO verify in a project where we use LambdaTest
```
Comment (mateuszwojtczak) [inline packages/patrol/android/src/main/kotlin/pl/leancode/patrol/LambdaTestPatrolJUnitRunner.java:22]: remove this TODO

**Review by mateuszwojtczak — CHANGES_REQUESTED** [review]
Also, let's add some page to the docs in the "Integrations" section

File: packages/patrol/android/src/main/kotlin/pl/leancode/patrol/LambdaTestPatrolJUnitRunner.java (hunk 2 above)
Comment (Gauravpandeyr) [reply packages/patrol/android/src/main/kotlin/pl/leancode/patrol/LambdaTestPatrolJUnitRunner.java]: Removed

File: packages/patrol/example/android/app/build.gradle
Comment (Gauravpandeyr) [reply packages/patrol/example/android/app/build.gradle:1]: Not a change.

File: packages/patrol/example/android/app/src/androidTest/java/pl/leancode/patrol/example/MainActivityTest.java
Comment (Gauravpandeyr) [reply packages/patrol/example/android/app/src/androidTest/java/pl/leancode/patrol/example/MainActivityTest.java:1]: Not a change

File: packages/patrol/example/android/gradle.properties
Comment (Gauravpandeyr) [reply packages/patrol/example/android/gradle.properties:1]: Not a change

File: packages/patrol/example/android/settings.gradle
Comment (Gauravpandeyr) [reply packages/patrol/example/android/settings.gradle:1]: Not a change

**Review by Gauravpandeyr — COMMENTED** [review]
These were originally present in the repository, I have added them back. The only change is in the LambdaTestPatrolJUnitRunner.java file only. Everything else is exactly the same.

**Comment by Gauravpandeyr** [comment]
> Also, let's add some page to the docs in the "Integrations" section

Added the doc:
https://github.com/leancodepl/patrol/pull/2424/files#diff-79e29b3af6a95e45d7a1250a3b5e23db3513950c6fed17874cef02a5dc15ba2

**Pull request closed without merging**